pkg/npm/npm.go

```diff
@@ -354,23 +356,28 @@ func (exec *Execute) checkIfLockFilesExist() (bool, bool, error) {
 // CreateBOM generates BOM file using CycloneDX from all package.json files
 func (exec *Execute) CreateBOM(packageJSONFiles []string) error {
 	execRunner := exec.Utils.GetExecRunner()
-	// Install CycloneDX Node.js module locally without saving in package.json
-	err := execRunner.RunExecutable("npm", "install", "@cyclonedx/bom@^3.10.6", "--no-save")
+	// Install CycloneDX Node.js module via npx without saving in package.json / polluting globals
+	// See https://github.com/CycloneDX/cyclonedx-node-npm#installation
+	err := execRunner.RunExecutable("npx", "--package", cycloneDxPackageVersion, "--call", "exit")
 	if err != nil {
-		return err
+		return fmt.Errorf("failed to install cycloneDx package: %w", err)
 	}
 
 	if len(packageJSONFiles) > 0 {
 		for _, packageJSONFile := range packageJSONFiles {
 			path := filepath.Dir(packageJSONFile)
 			params := []string{
-				"cyclonedx-bom",
-				path,
-				"--output", filepath.Join(path, npmBomFilename),
+				cycloneDxPackageVersion,
```

Comment:
This compiles to `npx @cyclonedx/cyclonedx-npm@1.11.0 ..`, I guess the package name would be enough?!

Reply:
Since we are using npc, If we don't specify version number , npx looks for latest version by default

Reply:
Ok, and this is all about pinning it down, understood.